python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,66 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_creds](#create_creds) | Create Credentials|
```

Comment:
Perhaps call this something closer to: "create_application_credentials"

Reply:
Or maybe more neutrally "service_account" to make it more cloud native.

Reply:
@ellis-bigelow how about `create_credentials`?  Personally I think the `create_application_credentials` is too long for user inputting, and in fact we ceated secret and service_account, seems that's not best to call `service_account`.

Reply:
Updated to `create_credentials`, thanks.